### PR TITLE
add rallly

### PR DESCRIPTION
**Pull request by luclu**
https://github.com/lukevella/rallly

### CODE REVIEW

File: apps/rallly/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+ENV \
+    DATABASE_URL=postgres://postgres:postgres@localhost:5432/db \
+    NEXT_PUBLIC_BASE_URL=http://localhost:3000 \
+    SECRET_PASSWORD=minimum-32-characters \
+    SUPPORT_EMAIL=foo@yourdomain.com \
+    SMTP_HOST=your-smtp-server \
+    SMTP_PORT=587 \
+    SMTP_SECURE=false \
+    SMTP_USER=your-smtp-user \
+    SMTP_PWD=your-smtp-password
```
Comment (onedr0p) [inline apps/rallly/Dockerfile:27]: Please only include bare minimum required env vars needed for the application to run

File: apps/rallly/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+|----------------------|--------------------------------------------------|
+| NEXT_PUBLIC_BASE_URL | `http://localhost:3000`                          |
+| DATABASE_URL         | `postgres://postgres:postgres@localhost:5432/db` |
+| SECRET_PASSWORD      | `minimum-32-characters`                          |
+| SUPPORT_EMAIL        | `foo@yourdomain.com`                             |
+| SMTP_HOST            | `your-smtp-server`                               |
+| SMTP_PORT            | `587`                                            |
+| SMTP_SECURE          | `false`                                          |
+| SMTP_USER            | `your-smtp-user`                                 |
+| SMTP_PWD             | `your-smtp-password`                             |
```
Comment (onedr0p) [inline apps/rallly/README.md:17]: This section can be delete as it is not a custom configuration we are scaffolding with `envsubt`. I expect people to be able to do research and look up env vars if they are properly supported by the application itself.

File: apps/rallly/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+## Additional requirements
+In your kubernetes manifest override CMD with `sh -c "yarn prisma migrate deploy --schema prisma/schema.prisma && yarn start"`.
```
Comment (onedr0p) [inline apps/rallly/README.md:20]:
```suggestion
## Nuances

In your kubernetes manifest override CMD with `sh -c "yarn prisma migrate deploy --schema prisma/schema.prisma && yarn start"`.
```

File: apps/rallly/metadata.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+      "name": "main",
+      "platforms": [
+        "linux/amd64"
+      ],
+      "stable": false,
+      "tests": {
+        "enabled": true,
+        "type": "web"
+      }
+    }
```
Comment (onedr0p) [inline apps/rallly/metadata.json:26]: Any reason why you have 2 channels here? In most cases we should just be building the most stable version of the app.

**Review by onedr0p — CHANGES_REQUESTED** [review]

File: apps/rallly/Dockerfile (hunk 1 above)
Comment (onedr0p) [reply apps/rallly/Dockerfile]: Are the others required for the app to start?

File: apps/rallly/metadata.json [hunk 5]
```diff
@@ -0,0 +1,17 @@
+{
+  "app": "rallly",
+  "base": false,
+  "channels": [
+    {
+      "name": "production",
```
Comment (onedr0p) [inline apps/rallly/metadata.json:6]:
```suggestion
      "name": "stable",
```

Comment (onedr0p) [reply apps/rallly/metadata.json:6]: Not necessarily, it can just be stable when there's one channel

Comment (onedr0p) [reply apps/rallly/metadata.json:6]: Sorry I haven't forgot about this, just been busy.

Comment (onedr0p) [reply apps/rallly/metadata.json:6]: Is there any confirmation on the production branch being updated, should we just use main instead?

Comment (samip5) [reply apps/rallly/metadata.json:6]: > Is there any confirmation on the production branch being updated, should we just use main instead?

There has been "The main branch is considered production and is what's available on rallly.co."

**Comment by samip5** [comment]
Please remember arm64 platform too. :)

File: apps/rallly/metadata.json (hunk 5 above)
Comment (onedr0p) [reply apps/rallly/metadata.json:6]: We can just hardcode "stable" in the metadata.json and main in the latest and Dockerfile scripts. I don't foresee long-lived branches that will need to be built and ran.

**Review by onedr0p — APPROVED** [review]

**Pull request merged**